src/shared/components/title-fades/title-fades.component.scss

```diff
@@ -1,96 +1,89 @@
-.upcoming-events-title-orange{
-    position: relative;
-    overflow: hidden;
+.upcoming-events-title-orange {
+  position: relative;
+  overflow: hidden;
+  height: 32px;
+  background: linear-gradient(270deg, transparent, rgba(255, 116, 24, 0.8) 20%);
+  h2 {
+    padding-top: 7px;
     height: 32px;
-    background: linear-gradient(270deg, transparent , #FF7418 20%);
-    h2{
-        padding-top: 7px;
-        height: 32px;
-        font-weight: 700;
-        font-size: 18px;
-        padding-left: 80px;
-        line-height: 21.6px; 
-        color: #0D0D0D;
-        position: relative;
-
-     }
-     .first-fade{
-        border: 9px solid black;
-        height: 200px;
-        width: 200px;
-        position: absolute;
-        transform: translateY(-50%) rotate(45deg);
-        top: 50%;
-        right: 92%;
-     }
-
-     .second-fade{
-        border: 9px solid black;
-        height: 200px;
-        width: 200px;
-        position: absolute;
-        transform: translateY(-50%) rotate(45deg);
-        top: 50%;
-        right: 98%;
-     }
+    font-weight: 700;
+    font-size: 18px;
+    padding-left: 80px;
+    line-height: 21.6px;
+    color: #0d0d0d;
+    position: relative;
+  }
+  .first-fade {
+    border: 9px solid black;
+    height: 200px;
+    width: 200px;
+    position: absolute;
+    transform: translateY(-50%) rotate(45deg);
+    top: 50%;
+    right: 92%;
+  }
 
+  .second-fade {
+    border: 9px solid black;
+    height: 200px;
+    width: 200px;
+    position: absolute;
+    transform: translateY(-50%) rotate(45deg);
+    top: 50%;
+    right: 98%;
+  }
 }
 
-
-
 // If needed change the background
-.upcoming-events-title-yellow{
-   position: relative;
-   overflow: hidden;
-   height: 32px;
-   background: linear-gradient(270deg, transparent , #FFDC23 20%);
-   h2{
-       padding-top: 7px;
-       height: 32px;
-       font-weight: 700;
-       font-size: 18px;
-       padding-left: 80px;
-       line-height: 21.6px; 
-       color: #0D0D0D;
-       position: relative;
-
-    }
-    .first-fade{
-       border: 9px solid black;
-       height: 200px;
-       width: 200px;
-       position: absolute;
-       transform: translateY(-50%) rotate(45deg);
-       top: 50%;
-       right: 92%;
-    }
-
-    .second-fade{
-       border: 9px solid black;
-       height: 200px;
-       width: 200px;
-       position: absolute;
-       transform: translateY(-50%) rotate(45deg);
-       top: 50%;
-       right: 98%;
-    }
+.upcoming-events-title-yellow {
+  position: relative;
```

Comment:
Aren't the styles up to background the same with the ".....title-orange" class?

Reply:
Then i set class for titles positioning and them set class only for color change it appear other problem. The class yellow-title is applied conditionally based on the value of the color variable, but it might not directly inherit the styles from the .upcoming-events class due to the use of Angular's dynamic class assignment. To remove 3 lines from the css i need to change the logic and add 2 lines in TS file. In this case i think that will be overkill.